net/udp-broadcast-relay-redux-openwrt/Makefile

```
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udp-broadcast-relay-redux
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
```

```suggestion
PKG_RELEASE:=$(AUTORELEASE)
```
Just use `$(AUTORELEASE)` for simplicity.
And calling `$(PKG_SOURCE_VERSION)` here won't work, as it isn't defined before referred.